src/pages/minjiJeong/Main/Comment/Comment.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import CommentList from './CommentList';
+import './Comment.scss';
+
+let id = 4;
+
+class Comment extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentList: [],
+      content: '',
+      isLiked: false,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({ commentList: this.props.commentList });
+  }
+
+  getComment = e => {
+    this.setState({
+      content: e.target.value,
+    });
+  };
+
+  uploadComment = e => {
+    this.state.commentList.push({
+      id: id,
+      userId: '_minji.jeong',
+      content: this.state.content,
+      isLiked: false,
+    });
```

Comment:
- React의 state값은 불변하게 다뤄야 합니다(react에서 state의 변경 여부를 판단할 때의 동작과 연관되어있습니다)
- Array.push, Array.unshift 등의 원본 배열을 변경하는 메소드나, 객체의 속성값을 직접 변경하는 경우는 객체나 배열을 가변하게 사용한 것이기 때문에 좋지 않습니다.
- map, filter, concat 등 새로운 배열을 만들어내는 함수를 통해서 새로운 배열을 만들고 그 배열을 새로운 state에 할당하는 식으로 진행해주세요
```suggestion
    const newCommentList = this.state.commentList.concat({
      id: id,
      userId: '_minji.jeong',
      content: this.state.content,
      isLiked: false,
    });
    
    this.setState({commentList:newCommentList, content:""})
```

Reply:
네! 이부분도 주의하겠습니다 👀 수정하였습니다!